### PR TITLE
✨ Added new functions and fixed a few thing.

### PR DESCRIPTION
**Pull request by CroxyTheDev**

### CODE REVIEW

**Comment by acarnd03** [comment]
`<Structure>.method()` is forbidden, lost performance.

File: src/structures/createChannel.js [hunk 1]
```diff
@@ -33,6 +35,14 @@ const createChannel = ((channelData) => {
...
     channel.parentID = channelData.parent_id || null;
   }
+  
+  channel.createMessage = (mesaj) => {
+    sendMessage(channelData.id, mesaj)
+  }
+
+  channel.delete = () => {
+    deleteChannel(channelData.id)
+  }
```
Comment (acarnd03) [inline src/structures/createChannel.js:45]: ups

File: src/structures/createMessage.js [hunk 2]
```diff
@@ -91,8 +95,20 @@ const createMessage = (async(messageData) => {
...
+     return pinMessage(messageData.channel_id, messageData.id)
+  }
+  
+  message.unpin = () => {
+     return unpinMessage(messageData.channel_id, messageData.id)
+  }
+  
+  message.delete = () => {
+     return deleteMessage(messageData.channel_id, messageData.id)
+  }
```
Comment (acarnd03) [inline src/structures/createMessage.js:109]: ups

**Review by acarnd03 — COMMENTED** [review]
another thing, requests must be async for stability, `request(method, path)` is valid yea but it is unstable code.

**Comment by CroxyTheDev** [comment]
Edited too many codes. Please review again.

**Pull request merged**